game-core/src/main/java/games/strategy/engine/auto/health/check/LocalSystemChecker.java

```diff
@@ -1,24 +1,44 @@
-package games.strategy.engine.framework.systemcheck;
+package games.strategy.engine.auto.health.check;
 
 import java.io.File;
 import java.io.IOException;
 import java.net.URL;
 import java.net.URLConnection;
+import java.util.Collection;
+import java.util.Optional;
 import java.util.Set;
 import java.util.stream.Collectors;
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.collect.ImmutableSet;
 
+import games.strategy.debug.ClientLogger;
+
 /**
  * This class runs a set of local system checks, like access network, and create a temp file.
  * Each check is always run, and this class records the results of those checks.
  */
 public final class LocalSystemChecker {
 
+  public static void launch() {
```

Comment:
It is because it is static, all statics tend to go first.

Reply:
Ah, didn't realize that was our convention.  We should probably codify that with the Checkstyle DeclarationOrder rule if it supports customization beyond the [Java code conventions](http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141855.html#1852).

Reply:
Following up on this...  The Checkstyle [DeclarationOrder rule](http://checkstyle.sourceforge.net/config_coding.html#DeclarationOrder) only supports the declaration order defined in the Java code conventions and can only be customized to ignore certain parts of those conventions.  It cannot be customized to support user-defined declaration orders, such as all static members first.

So, should devs be using the static-first declaration order going forward?  Since we can't enforce it, should it at least be in the dev docs?  How should it play with the existing convention to use depth-first ordering (e.g. an instance method that calls a static method that is otherwise unused anywhere else)?

Reply:
This is a good conversation to finish, let's follow up in: https://github.com/triplea-game/triplea/issues/3231